app/javascript/retrospring/features/announcement/index.ts

```diff
@@ -0,0 +1,14 @@
+import registerEvents from 'utilities/registerEvents';
+import closeAnnouncementHandler from './close';
+
+export default (): void => {
+  registerEvents([
+    { type: 'click', target: document.querySelector('.announcement button.close'), handler: closeAnnouncementHandler },
+  ]);
+
+  document.querySelectorAll('.announcement').forEach(function (el: HTMLDivElement) {
+    if (!window.localStorage.getItem(el.dataset.announcementId)) {
```

Comment:
```suggestion
    if (!window.localStorage.getItem(`announcement${el.dataset.announcementId}`)) {
```

without this, it queries for a key `1` in Local Storage